File: app/src/main/java/com/amusoft/gdgfirechat/FirechatAplication.java
package com.amusoft.gdgfirechat;

import android.app.Application;

import com.google.firebase.FirebaseApp;
import com.google.firebase.database.FirebaseDatabase;
import com.google.firebase.messaging.FirebaseMessaging;

/**
 * Created by dice on 6/10/16.
 */
public class FirechatAplication extends Application {
    FirebaseDatabase myFirebaseRef;

    @Override
    public void onCreate(){
        super.onCreate();

        if (!FirebaseApp.getApps(this).isEmpty()) {
            FirebaseDatabase.getInstance().setPersistenceEnabled(true);

            FirebaseMessaging.getInstance().subscribeToTopic("android");
        }



    }
}

File: app/src/main/java/com/amusoft/gdgfirechat/FirechatAplication.kt
package com.amusoft.gdgfirechat

import androidx.multidex.MultiDexApplication
import com.google.firebase.FirebaseApp
import com.google.firebase.database.FirebaseDatabase
import com.google.firebase.messaging.FirebaseMessaging

/**
 * Created by Tobenna on 13/08/19
 */
class FirechatAplication : MultiDexApplication() {

    override fun onCreate() {
        super.onCreate()

        if (FirebaseApp.getApps(this).isNotEmpty()) {
            FirebaseDatabase.getInstance().setPersistenceEnabled(true)

            FirebaseMessaging.getInstance().subscribeToTopic("android")
        }


    }
}
